### DISCUSSION TITLE
Where to change keybinding for starting cheatsheet.

### DISCUSSION THREAD

**Discussion by krister-dahl** [end-4/dots-hyprland#405] [Q&A] [answered]
As the title says. Where to change the keybinding super + / for viewing the cheatsheet. Cant make this combo work on a Swedish keyboard.

**Comment by end-4** [comment, marked as answer]
in `~/.config/hypr/hyprland/keybinds.conf` search for "cheatsheet" and modify that line

**Reply by krister-dahl** [reply]
Great! Thanks! 😀 Just tried to install your dotfiles from within Endeavour OS. When launching Hyprland the cursor is upside down. Any clues?

**Reply by end-4** [reply]
no idea

**Reply by krister-dahl** [reply]
Sorry, everything was fixed by a reboot 😀
